tests/backup/backup_sse_test.go

```diff
@@ -0,0 +1,424 @@
+package tests
+
+import (
+	"fmt"
+	. "github.com/onsi/ginkgo"
+	"github.com/pborman/uuid"
+	api "github.com/portworx/px-backup-api/pkg/apis/v1"
+	"github.com/portworx/sched-ops/k8s/core"
+	"github.com/portworx/sched-ops/k8s/storage"
+	"github.com/portworx/torpedo/drivers/backup"
+	"github.com/portworx/torpedo/drivers/scheduler"
+	"github.com/portworx/torpedo/drivers/scheduler/k8s"
+	"github.com/portworx/torpedo/pkg/log"
+	. "github.com/portworx/torpedo/tests"
+	v1 "k8s.io/api/core/v1"
+	storageApi "k8s.io/api/storage/v1"
+	metaV1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"strings"
+	"sync"
+	"time"
+)
+
+var _ = Describe("{sseS3encryption}", func() {
+
+	// testrailID corresponds to: https://portworx.testrail.net/index.php?/cases/view/93014
+	var (
+		scheduledAppContexts                 []*scheduler.Context
+		backupLocationUID                    string
+		cloudCredUID                         string
+		cloudCredUidForBlWithoutSse          string
+		backupsWithSse                       []string
+		backupsWithOutSse                    []string
+		cloudCredUidList                     []string
+		customBackupLocationWithSse          string
+		customBucketWithOutPolicy            string
+		scheduleName                         string
+		backupLocationUidWithoutSse          string
+		backupLocationWithoutSse             string
+		credName                             string
+		credNameForBlWithoutSse              string
+		customBucketsWithDenyPolicy          []string
+		bkpNamespaces                        []string
+		sourceScName                         *storageApi.StorageClass
+		scCount                              int
+		scNames                              []string
+		clusterStatus                        api.ClusterInfo_StatusInfo_Status
+		clusterUid                           string
+		restoreList                          []string
+		backupNames                          []string
+		schedulePolicyName                   string
+		schedulePolicyUid                    string
+		latestScheduleBackupName             string
+		backupName                           string
+		appContextsToBackup                  []*scheduler.Context
+		backupNameAfterRemovalOfDenyPolicy   string
+		newBackupLocationWithSseAfterRestart string
+		backupNameAfterPxBackupRestart       string
+		customBuckets                        []string
+	)
+
+	storageClassMapping := make(map[string]string)
+	namespaceMap := make(map[string]string)
+	params := make(map[string]string)
+	k8sStorage := storage.Instance()
+	backupLocationMap := make(map[string]string)
+
+	JustBeforeEach(func() {
+		StartTorpedoTest("sseS3encryption",
+			"Backup and restore from Backup location created with SSE-S3 and deny policy set from aws s3", nil, 93014)
+		log.InfoD("Deploy applications needed for backup")
+		scheduledAppContexts = make([]*scheduler.Context, 0)
+		numberOfNameSpace := 4
```

Comment:
Please follow the camel case correctly. A better variable name would be `namespaces` or `scaleFactor`.
`NameSpace` is not the right way. It should be `Namespace`.